File: src/main/java/com/appointmed/dto/DoctorResponse.java
package com.appointmed.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class DoctorResponse {
    private Long id;
    private Long userId;
    private String name;
    private String email;
    private String specialization;
    private String licenseNumber;
    private String phone;
    private String clinicAddress;
    private String avatarUrl;
    private boolean available;
    private double rating;
    private int reviews;
}

File: mobile/app/src/main/java/com/appointmed/mobile/data/model/DoctorResponse.kt
package com.appointmed.mobile.data.model

data class DoctorResponse(
    val id: Long = 0,
    val userId: Long = 0,
    val name: String = "",
    val email: String = "",
    val specialization: String = "",
    val licenseNumber: String? = null,
    val phone: String? = null,
    val clinicAddress: String? = null,
    val avatarUrl: String? = null,
    val available: Boolean = true,
    val rating: Double = 0.0,
    val reviews: Int = 0,
    val consultationFee: Double = 0.0
)
